.github/workflows/coverage.yml

```diff
@@ -0,0 +1,48 @@
+name: Coverage Testing
+on: push
+
+jobs:
+  go_coverage:
+    name: Go Coverage Test
+    defaults:
+      run:
+        working-directory: ./ff/serving
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - name: Install grpc_tools
+        run: pip install grpcio-tools
+
+      - name: Install Protobuf
+        run: sudo snap install protobuf --classic
+
+      - name: Setup Proto
+        run: ./gen_grpc.sh
+
+      - name: Build
+        run: go build
```

Comment:
some builds are failing in the github actions, should it be sure to terminate the whole job if this happens?

Reply:
Confirmed that any step that fails will cancel entire job. The exception right now is Go tests do to upcoming rework